app/views/admin/articles/_form.html.haml

```diff
@@ -0,0 +1,12 @@
+= form_with(
+      model: [:admin, article],
+      url: article.new_record? ? admin_articles_path : admin_article_path(article),
+      local: true,
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder
+    ) do |form|
+  = form.govuk_error_summary order: %i[title markdown]
+  = form.govuk_text_field :title 
+  = form.govuk_text_area :markdown
+  = form.hidden_field :category, value: 'article'
+  = form.hidden_field :position, value: 9999
```

Comment:
As above, I'd keep assigning positions, you don't have to use them.  Then you don't have to have the special cases when checking for position in the rest of the code